programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -2952,6 +3014,7 @@ static int add_identity_opts(int argc, char *argv[]) {
 static CommandLine enroll_cmd = make_command("enroll", "enroll Ziti identity",
         "-j|--jwt <enrollment token> -i|--identity <identity> [-k|--key <private_key> [-c|--cert <certificate>]] [-n|--name <name>]",
         "\t-j|--jwt\tenrollment token file\n"
+        "\t-x|--proxy [username[:password]@]hostname_or_ip:port\tproxy to use when connecting to OpenZiti controller"
```

Comment:
we should probably change this to allow other proxy types in the future (socks, socks5)
add `proto://` in front
then you can check proto to select proxy implementation (error/reject anything other than `http` now)

Reply:
done